README.md

```diff
@@ -124,6 +126,48 @@ The schedule of the garbage collection, there can always only be one.
 }
 ```
 
+### retention-policies.json
+
+Definition of the retention policies.
+The retention policies can be set per project.
+They can be found in each project page under the tab Policy.
+
+```json
+[
+  {
+    "scope": {
+      "level": "project",
+      "ref": 2
```

Comment:
Since we need a reference here for the project:
Would it make sense to add to the description something like: "This adds a retention policy for the project with ID 2. The project ID can be found via ..."?

It also looks like that the examples in the readme may lack some parts which we configure on our Harbor. E.g. IDs on registries. Maybe we want to update the readme and also add hints on special parts like IDs etc.? But this is probably an own issue. What do you think?

Reply:
I added instructions for finding and setting project ids and registry ids in the `README.md`.